pilot/pkg/secrets/kube/multicluster.go

```diff
@@ -36,17 +37,44 @@ type Multicluster struct {
 
 var _ secrets.MulticlusterController = &Multicluster{}
 
+var (
+	clusterType = monitoring.MustCreateLabel("cluster_type")
+
+	clustersCount = monitoring.NewGauge(
+		"istiod_managed_clusters_total",
```

Comment:
Should we remove the _total suffix? I think it is meant for counter.

Reply:
done.